worker-pools.yml

```diff
@@ -250,6 +253,10 @@ pools:
       # the "t2a" machine series is only available in the "a", "b", and "f" zones of the "us-central1" region.
       regions: [us-central1]
       image: monopacker-ubuntu-2204-wayland-arm64
+      security:
+        by-chain-of-trust:
+          trusted: trusted
+          default:
```

Comment:
This seems unused AFAICT?

Reply:
Weird! This was failing some check as I was making these changes. But seems like I changed something else that made the error go away.